src/bci_build/package/busybox/README.md.j2

```diff
@@ -1,5 +1,11 @@
 # SLE BCI-BusyBox: the smallest and GPLv3-free image
+{% include 'badges.j2' %}
 
-The SLE BCI-BusyBox image comes with the most basic tools provided by the BusyBox project. The image contains no GPLv3 licensed software. When using the image, keep in mind that there are differences between the BusyBox tools and the GNU Coreutils. This means that scripts written for a system that uses GNU Coreutils may require modification to work with BusyBox.
+The SLE BCI-BusyBox image comes with the most basic tools provided by the BusyBox project.
```

Comment:
do you mean something like "most basic tools provided by Coreutils" ? or a really concrete list? the concrete list is subject to change, it would be maintenance effort to keep this up to date with busybox updates that potentially introduce new things.

Reply:
> The SLE BCI-BusyBox image comes with tiny versions of many common UNIX utilities provided by the BusyBox project.

This is more in line with what BusyBox is in reality, I took this bit from "BusyBox combines tiny versions of many common UNIX utilities into a single small executable."